test/plugin.test.js

```diff
@@ -337,3 +337,21 @@ QUnit.test('custom error details should override defaults', function(assert) {
   assert.strictEqual(document.querySelector('.vjs-errors-message').textContent,
     customError.message, 'message should match custom override value');
 });
+
+QUnit.test('Append Flash error details when flash is not supported', function(assert) {
+  let isFlashSupported = videojs.getComponent('Flash').isSupported();
+
+  if (!isFlashSupported) {
```

Comment:
we want this test to run regardless of wether the environment supports flash or not. This probably means capturing and replacing `videojs.getComponent('Flash').isSupported` with something like `() => false` and then setting `videojs.getComponent('Flash').isSupported` back to the old value at the bottom of the function.